modules/machine-health-checks-resource.adoc

```diff
@@ -39,9 +40,53 @@ format. For example, `prod-node-us-east-1a`.
 <4> Specify the timeout duration for a node condition. If a condition is met for the duration of the timeout, the Machine will be remediated. Long timeouts can result in long periods of downtime for the workload on the unhealthy Machine.
 <5> Specify the amount of unhealthy machines allowed in the targeted pool of
 machines. This can be set as a percentage or an integer.
+<6> Specify the timeout duration that a MachineHealthCheck must wait for a node to join the cluster before a Machine is determined to be unhealthy.
 
 [NOTE]
 ====
 The `matchLabels` are examples only; you must map your machine groups based on
 your specific needs.
 ====
+
+[id="machine-health-checks-short-circuiting_{context}"]
+== Short-circuiting MachineHealthCheck remediation
+
+To ensure that MachineHealthChecks remediate Machines only when the cluster is healthy, short circuiting is implemented to prevent further remediation.
+Short-circuiting is configured through the `maxUnhealthy` field in the MachineHealthCheck resource.
+
+If the user defines a value for the `maxUnhealthy` field,
+before remediating any Machines, the MachineHealthCheck compares the value of `maxUnhealthy`
+with the number of Machines within its target pool that it has determined to be unhealthy.
+If the number of unhealthy Machines exceeds the limit set by `maxUnhealthy`, remediation is not performed.
+
+[IMPORTANT]
+====
+If `maxUnhealthy` is not set, the value defaults to `100%`.
+Therefore, the short-circuiting mechanism is disabled by default and Machines are remediated regardless of the state of the cluster.
+====
+
+The `maxUnhealthy` field can be set as either an integer or percentage.
+There are different remediation implementations depending on the `maxUnhealthy` value.
+
+=== With an absolute value
```

Comment:
I think it would be useful to the reader and for findability if this heading included more context. For example:

"=== Setting `maxUnhealthy by using an absolute value"